src/activity/activity.ts

```diff
@@ -1,8 +1,5 @@
-import { Logger } from "../shared/utils";
 import { Agreement, ProviderInfo } from "../market/agreement";
 import { EventEmitter } from "eventemitter3";
-import { ExecutionOptions, ExeScriptExecutor } from "./exe-script-executor";
-import { IActivityApi } from "./types";
 
 /** @deprecated */
```

Comment:
this can already be removed

Reply:
already removed in other PR: https://github.com/golemfactory/golem-js/pull/958/commits/4a7659ece0c271f8384faefb11be41865fc4b4b7